css-basics_social-card/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "html-and-the-web_div-salad",
```

Comment:
Just curious why was this field changed?

Reply:
good question ... :)